### DISCUSSION TITLE
Disabled button in markdown editor

### DISCUSSION THREAD

**Discussion by MDesharnaisX** [requarks/wiki#3752] [Help / Questions] [answered]
Hi everybody,

In my markdown editor, I have disabled buttons :
![image](https://user-images.githubusercontent.com/78907460/112279995-d316f180-8c84-11eb-84bb-ea7c047a2bd7.png)
Would somebody know what they are and why they are disabled ? (Missing extension or unavailable feature for this editor)

Thanks

**Comment by NGPixel** [comment, marked as answer]
These are features that will be available in future versions.
